.gitignore

```diff
@@ -28,5 +28,3 @@ node_modules
 
 .DS_Store
 *.iml
-
-dist
```

Comment:
I'd love to avoid checking in generated code if possible. Maybe publishing the dist artefacts as a part of a GitHub release instead?